hera_mc/weather.py

```diff
@@ -189,19 +196,22 @@ def _helper_create_from_sensors(starttime, stoptime, variables=None):
                 timestamp = item.timestamp
             # sometimes there are duplicates. Protect against that
             if timestamp not in time_vals.keys():
-                time_vals[timestamp] = float(item.value)
-
-        reduction = weather_sensor_dict[variable]['reduction']
-        period = weather_sensor_dict[variable]['period']
-
-        times_use, values_use = _reduce_time_vals(np.array(time_vals.keys()),
-                                                  np.array(time_vals.values()),
-                                                  period, strategy=reduction)
-
-        for count, timestamp in enumerate(times_use.tolist()):
-            time_obj = Time(timestamp, format='unix')
-            weather_obj_list.append(WeatherData.create(time_obj, variable,
-                                                       values_use[count]))
+                time_vals[timestamp] = item.value
+                value_times.append(timestamp)
+                sensor_values.append(float(item.value))
```

Comment:
Did you switch to lists because you actually saw a problem? I had started with lists but switched to dicts at Peter's suggestion. If you're worried that the key and value won't match because of the order not being guaranteed, see this: https://stackoverflow.com/questions/835092/python-dictionary-are-keys-and-values-always-the-same-order

Reply:
As far as I could tell, the resampling schemes need monotonically (essentially) increasing values.  If the dict isn't doing anything else I can remove it.  In the random-ish key order, the resampling fails (or can -- it worked for me sometimes, but sometimes not).

Reply:
Ah, I think when I wrote it in the first place I was just doing the decimate and that doesn't require that they're ordered, but the others might. If there's an example when it fails, we should add it to the tests to make sure it works. If ordering is an issue, I'm fine with either using lists in the first place or sorting them afterwards.